core/control-panel/impl/src/controllers/station.rs

```diff
@@ -130,7 +131,16 @@ impl StationController {
     async fn deploy_station(&self, input: DeployStationInput) -> ApiResult<DeployStationResponse> {
         let ctx = CallContext::get();
         let _lock = STATE
-            .with(|state| CallerGuard::new(state.clone(), ctx.caller(), None))
+            .with(|state| {
+                CallerGuard::new(
+                    state.clone(),
+                    ctx.caller(),
+                    CallerGuardParams {
+                        max_concurrency: Some(MAX_DEPLOYED_STATIONS_PER_DAY),
```

Comment:
We should decrease MAX_DEPLOYED_STATIONS_PER_DAY by the current number of already deployed stations on the same day.